e2e/settings.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { expect, test } from '@playwright/test'
+import { electronTest } from './helpers'
+import { LegendaryCommand } from '../src/backend/storeManagers/legendary/commands'
+
+electronTest('Settings', async (app, page) => {
+  // stub `legendary --version`
+  await app.evaluate(({ ipcMain }) => {
+    ipcMain.emit(
+      'setRunLegendaryCommandStub',
+      function (command: LegendaryCommand) {
+        if (command['--version']) {
+          return {
+            stdout: 'legendary version "0.20.33", codename "Undue Alarm"',
+            stderr: ''
+          }
+        } else {
+          return {
+            stdout: '',
+            stderr: ''
+          }
+        }
+      }
+    )
+  })
```

Comment:
I tried to create a helper method to abstract this evaluate/emit code, but I keep running into this issue `Error: electronApplication.evaluate: ReferenceError: .... is not defined` because of how the evaluation context changes

I can add more details if needed